module-new/src/components/button/button.theme.css

```diff
@@ -12,95 +12,97 @@
   filter: none;
   opacity: 0.6;
 }
 
-.arm-button[data-status="positive"] {
+.arm-button[data-status='positive'] {
   background-color: var(--arm-color-positive);
 }
 
-.arm-button[data-status="negative"] {
+.arm-button[data-status='negative'] {
   background-color: var(--arm-color-negative);
 }
 
-.arm-button[data-status="warning"] {
+.arm-button[data-status='warning'] {
   background-color: var(--arm-color-warning);
 }
 
-.arm-button[data-status="info"] {
+.arm-button[data-status='info'] {
   background-color: var(--arm-color-info);
 }
 
-.arm-button[data-status="positive"][data-style="outline"] {
+.arm-button[data-status='positive'][data-style='outline'] {
   color: var(--arm-color-positive);
   border-color: var(--arm-color-positive);
 }
 
-.arm-button[data-status="negative"][data-style="outline"] {
+.arm-button[data-status='negative'][data-style='outline'] {
   color: var(--arm-color-negative);
   border-color: var(--arm-color-negative);
 }
 
-.arm-button[data-status="warning"][data-style="outline"] {
+.arm-button[data-status='warning'][data-style='outline'] {
   color: var(--arm-color-warning);
   border-color: var(--arm-color-warning);
 }
 
-.arm-button[data-status="info"][data-style="outline"] {
+.arm-button[data-status='info'][data-style='outline'] {
   color: var(--arm-color-info);
   border-color: var(--arm-color-info);
 }
 
-.arm-button[data-status="positive"][data-style="secondary"] {
+.arm-button[data-status='positive'][data-style='secondary'] {
   background-color: var(--arm-color-positive-secondary);
 }
 
-.arm-button[data-status="negative"][data-style="secondary"] {
+.arm-button[data-status='negative'][data-style='secondary'] {
   background-color: var(--arm-color-negative-secondary);
 }
 
-.arm-button[data-status="warning"][data-style="secondary"] {
+.arm-button[data-status='warning'][data-style='secondary'] {
   background-color: var(--arm-color-warning-secondary);
 }
 
-.arm-button[data-status="info"][data-style="secondary"] {
+.arm-button[data-status='info'][data-style='secondary'] {
   background-color: var(--arm-color-info-secondary);
 }
 
 /* STYLES */
 
-.arm-button[data-style="secondary"] {
+.arm-button[data-style='secondary'] {
   background-color: var(--arm-button-secondary-bg-color);
   color: var(--arm-button-secondary-fg-color);
 }
 
-.arm-button[data-style="outline"] {
+.arm-button[data-style='outline'] {
   background-color: transparent;
   border: var(--arm-button-outline-stroke-thickness) var(--arm-button-outline-stroke-color) solid;
   color: var(--arm-button-outline-stroke-color);
 }
 
-.arm-button[data-size="small"] {
+.arm-button[data-size='small'] {
   padding: var(--arm-spacing-xxsmall) var(--arm-spacing-medium);
   font-size: var(--arm-font-size-xsmall);
 }
 
-.arm-button[data-size="large"] {
+.arm-button[data-size='large'] {
   padding: var(--arm-spacing-medium) var(--arm-spacing-xxlarge);
   font-size: var(--arm-font-size-medium);
 }
 
 /* ICONS */
 
 .arm-button-left-icon {
+  display: flex;
```

Comment:
flexing these is too opinionated as it will directly alter the display type of children passed in by the consuming app, what's it achieving here?